indigo_app/static/javascript/indigo/utils.js

```diff
@@ -15,3 +15,21 @@ $(function() {
 
   $('body').on('click', 'a[data-confirm], button[data-confirm]', handleConfirm);
 });
+
+// Show popover when hovering on selected links
+$('body').on('mouseenter', 'a[data-popup-url]', function() {
+  var _this = this;
+  $.get(this.href + 'popup').then(function(html) {
+    $(_this).popover({content: html, html: true}).popover('show');
+  });
+  $('.popover').mouseleave(function () {
+    $(_this).popover('hide');
+  });
+}).on('mouseleave','a[href^="/works/"]:not(.btn)', function(){
```

Comment:
The scoping on this event should be `a[data-popup-url]` I think?

Reply:
Yes, it should. I forgot to update this.